packages/katello/katello-host-tools/katello-host-tools.spec

```diff
@@ -239,6 +247,7 @@ exit 0
 %else
 %attr(750, root, root) %{_sbindir}/katello-tracer-upload
 %endif
+%endif
```

Comment:
instead of wrapping them like this, any objections to using global variables for this?    Such as setting a  'build_tracer' variable at the top, and then using that down below.

Reply:
Np, but wouldn't it still be wrapped just the same? Want to make sure I'm not missing something

Reply:
it would still be wrapped, yeah just not based on whether its suse or not, but instead based on whether 'build_tracer' is set.  I think this help better define intent and makes things a bit clearer (I'll have to do more similar things when building this on suse)